ethsigner/signer/hashicorp/src/test/java/tech/pegasys/ethsigner/signer/hashicorp/HashicorpSubCommandTest.java

```diff
@@ -79,6 +105,28 @@ public void fullyPopulatedCommandLineParsesIntoVariables() {
     assertThat(string).contains(PORT);
     assertThat(string).contains(PATH_TO_SIGNING_KEY);
     assertThat(string).contains(FIFTEEN);
+
+    assertThat(hashiConfig.isTlsEnabled()).isTrue();
+    assertThat(hashiConfig.getTrustOptions()).isNotNull();
+    assertThat(hashiConfig.getTrustOptions().getPath()).isEqualTo(Path.of(CLIENT_CERT_PFX));
+    assertThat(hashiConfig.getTrustOptions().getPasswordFilePath())
+        .isEqualTo(Path.of(CLIENT_CERT_PASSWD));
+  }
+
+  @Test
+  public void tlsOptionalCommandLineParsesIntoVariables() {
```

Comment:
```suggestion
  public void commandLineWithTlsOptionsParsesIntoVariables() {
```

Reply:
reworded slightly differently to `commandLineWithTlsDisabledParsesIntoVariables`